### PR TITLE
[sunrisegaming-staking] Add sunrisegaming-staking strategy

### PR DESCRIPTION
**Pull request by sunrisedao**
Add sunrisegaming-staking strategy

- Total token in staking pool

### CODE REVIEW

File: src/strategies/sunrisegaming-staking/index.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import { multicall } from '../../utils';
+import { formatUnits } from '@ethersproject/units';
+import { BigNumber } from '@ethersproject/bignumber';
+
+export const author = 'sunrisegaming';
```
Comment (ChaituVR) [inline src/strategies/sunrisegaming-staking/index.ts:5]:
```suggestion
export const author = 'sunrisedao';
```

File: src/strategies/sunrisegaming-staking/index.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      {
+        internalType: 'uint256',
+        name: 'rewardDebt',
+        type: 'uint256'
+      }
+    ],
+    stateMutability: 'view',
+    type: 'function'
+  }
+];
```
Comment (ChaituVR) [inline src/strategies/sunrisegaming-staking/index.ts:38]:
```suggestion
const masterChefAbi = [
  'function userInfo(uint256, address) view returns (uint256 amount, uint256 rewardDebt)'
];
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Comment by sunrisedao** [comment]
@ChaituVR I've updated the source code

**Review by bonustrack — APPROVED** [review]

**Review by ChaituVR — APPROVED** [review]

**Comment by ChaituVR** [comment]
strategy is live: https://snapshot.org/#/strategy/sunrisegaming-staking

**Pull request merged**